MAUI/Cartesian-Charts/Get-the-touch-position-in-chart.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: Get the touch position in Syncfusion SfCartesianChart
+description: Get the touch position in SfCartesianChart
+platform: maui
+control: SfCartesianChart
+documentation: ug
+---
+
+# Get the touch position in SfCartesianChart
+
+ChartInteractiveBehavior provides the following override methods to get the x and y positions when touch the [`SfCartesianChart`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCartesianChart.html?tabs=tabid-1).
+
+* [`OnTouchUp`]() - Called when a user lifts their finger or release their touch input from the Chart area. 
+* [`OnTouchMove`]() - Called when a user's finger or touch input device is in contact within the Chart area and moves across it's surface.
+* [`OnTouchDown`]() -  Called when the user made the initial contact of a user's finger or touch input device with the Chart Area.
+
+{% tabs %}
+
+{% highlight c# %}
+
+    public class ChartInteractiveExt: ChartInteractiveBehavior
```

Comment:
@SuryaKaran2143 , How the user add this class in chart, i not find code for that.

Reply:
I include the XAML code in separate tab
It will useful to the user to add this class to chart